Add input for custom website build (#395)
product-os/flowzone: README.md

    - [`custom_test_matrix`](#custom_test_matrix)
    - [`custom_publish_matrix`](#custom_publish_matrix)
    - [`custom_finalize_matrix`](#custom_finalize_matrix)
    - [`cloudflare_website`](#cloudflare_website)

>     - [`custom_website_build`](#custom_website_build)

Reviewer: ```suggestion
    - [`docusaurus_website `](#docusaurus_website)
```

Default: `''`

#### `cloudflare_website`

Setting this to your existing CF pages project name will generate and deploy a website. Skipped if empty.

Type: _string_

Default: `""`

#### `docusaurus_website`

> Set to false to disable building docusaurus website. Set the build command in a npm script stored in `npm run deploy-docs`. Runs docusaurus builder if empty.

Reviewer: ```suggestion
Set to false to disable building a docusaurus website. If false the script `npm run deploy-docs` will be run if it exists.
```
